client/src/components/podcasts/PodcastTile.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const PodcastTile = ({ podcast }) => {
+  return (
+    <Link to={`/podcasts/${podcast.id}`}>
+      <div className="callout primary">
+        <h1>{podcast.name}</h1>
+        <p>{podcast.description}</p>
+      </div>
+    </Link>
+  );
+};
+
+export default PodcastTile;
```

Comment:
This also doesnt seem to be used in the show page so we can remove it from this PR